### PR TITLE
Add ability to specify direction for VolumeSlider

### PR DESCRIPTION
**Pull request by jonjomckay**
I added the appropriate documentation and tests too :)

### CODE REVIEW

File: docs/src/js/demos.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ import Prism from 'prismjs'
 import 'prismjs/components/prism-jsx.js'
 
 import * as rpc from '../../../dist/index.js'
+import { ControlDirection } from "../../../src";
```
Comment (alexanderwallin) [inline docs/src/js/demos.js]: We can just use `rpc.ControlDirection` here.

File: docs/src/sass/_controls.scss [hunk 2]
```diff
@@ -155,28 +155,48 @@ $color-active: fade-out($color-hover, 0.5);
...
 .VolumeSlider {
-  width: 8px;
-  height: 100px;
   // margin: 10px 0;
 
   &-value {
     position: absolute;
+  }
+
+  &.Direction-horizontal {
```
Comment (alexanderwallin) [inline docs/src/sass/_controls.scss]: Could we make these more like the `is*` classes? Perhaps `isHorizontal`/`isVertical` or `modHorizontal`/`modVertical`?

File: tests/VolumeSlider-test.js [hunk 3]
```diff
@@ -33,6 +33,34 @@ describe('<VolumeSlider />', () => {
...
+
+    const value = slider.find('.VolumeSlider-value')
+    const intent = slider.find('.VolumeSlider-intent')
+    const handle = slider.find('.VolumeSlider-handle')
+
+    expect(value.props().style).to.eql({ width: '60%' })
+    expect(intent.props().style).to.eql({ width: '20%' })
+    expect(handle.props().style).to.eql({ left: '60%' })
+  })
+
```
Comment (alexanderwallin) [inline tests/VolumeSlider-test.js]: Can we also add a test that the `direction` prop is passed on to `RangeControlOverlay`?

**Comment by jonjomckay** [comment]
@alexanderwallin thanks for the feedback - I made the changes!

**Comment by alexanderwallin** [comment]
Wonderful!

**Comment by alexanderwallin** [comment]
https://github.com/alexanderwallin/react-player-controls/releases/tag/v0.5.18

**Pull request merged**